Intersect.Client/Entities/Entity.cs

```diff
@@ -1061,6 +1127,13 @@ public virtual void Draw()
             }
         }
 
+        private int GetSpriteDirection()
+        {
+            int[] dTable = { 3, 0, 1, 2, 1, 2, 1, 2 };
```

Comment:
1. This is a constant, this should not be getting redefined on each invocation of this method.
2. Also don't even do this, because this is going to be bad for users. Sprite direction should be whichever cardinal direction the user had pressed first, and if both directions were pressed in the same frame it should prefer up and down instead of left/right.

Reply:
prefering up and down instead of left/right (TLOZ: A link to the past like diagonal movement) do _work_ fine when walking but creates a visual inconsistensy where we can't tell which is the current direction of the player sometimes, for example, is the player facing down, down left or down right?? we can't tell anymore visually by doing this and it leads to very confusing scenarios if we are switching targets that are diagonally placed.

ie: player targets and kills a diagonally positioned entity (down right), then the user keeps smashing the attack key since the character is "looking down" visually but then, the user will see how the player won't attack down since the last direction was down-right.... the only way a user could know this is by knowing how things work within the code 🙅🏽 but if we visually draw the character right, the user can tell for sure that the player is facing other direction than down and will procceed to either target the entity below or change their direction down manually.

Reply:
(TLOZ: A link to the past like diagonal movement)

![imagen](https://user-images.githubusercontent.com/17498701/216111267-d1b2e38d-e402-48ed-a21b-5385fd79a672.png)

![imagen](https://user-images.githubusercontent.com/17498701/216111295-b3ec1563-3faa-44ca-b578-40ac3fc5d971.png)

Reply:
https://user-images.githubusercontent.com/17498701/216145828-ed99e679-227d-4c9d-9b1b-80304329407e.mp4